src/Core/src/Handlers/Editor/EditorHandler.Windows.cs

```diff
@@ -61,6 +61,9 @@ protected override void DisconnectHandler(TextBox nativeView)
 		public static void MapHorizontalTextAlignment(EditorHandler handler, IEditor editor) =>
 			handler.NativeView?.UpdateHorizontalTextAlignment(editor);
 
+		[MissingMapper]
+		public static void MapCursorPosition(IViewHandler handler, ITextInput editor) { }
```

Comment:
It has been a long time since we started and now there is the windows variants of the selection mappers. So we can implement the `MapCursorPosition` and `MapSelectionLength` mappers now.

Reply:
Added these now